### ISSUE TITLE
FileSet show page should have a link to "Analytics"

### ISSUE THREAD

**Issue by mjgiarlo** [projecthydra/sufia#1388]
Confirm that https://github.com/projecthydra/sufia/blob/master/spec/views/stats/file.html.erb_spec.rb is linked to from the FileSet show view and tested in a view spec.

https://github.com/projecthydra/sufia/blob/master/app/views/generic_files/_show_actions.html.erb#L5-L8

*Migrated from https://github.com/projecthydra-labs/sufia-core/issues/167*

**Comment by barmintor** [comment]
@mjgiarlo We need to test whether this is satisfied generally by:
https://github.com/projecthydra/sufia/blob/242efd0a53b88ddaca82b7c7c637264de1f872a4/app/views/curation_concerns/base/_show_actions.html.erb#L3-L5

**Comment by mjgiarlo** [comment]
@barmintor That looks about right to me, though does that mean works and filesets will have analytics pages? I'm not sure we have sufficient plumbing to make that be the case at this point.

I believe we may have also discussed this in the context of migration paths and concluded that, if we need to choose between the work and fileset, the work should hold analytics (under the assumption that a GF *becomes* a Work and a new FileSet is *created* to hold the GF's binary content, i.e., the Work takes the GF's identifier so that routes can be setup to transparently 301 from /files/ID to the new Work URI). Though this is complicated because our analytics strategy current includes both pageviews (useful for both filesets and works) and downloads (useful only for filesets, though these numbers could be summed for Work-level download stats).

In short... there's this ticket ("fix the show page") and then there's the all the other work that this ticket may be hiding.

**Comment by grosscol** [comment]
I was under the impression that they were both going to have analytics with:
Work having page views
FileSet having downloads

**Comment by awead** [comment]
I agree with @grosscol

**Comment by mjgiarlo** [comment]
@awead Hrmm, how is what @grosscol said different from what I said?

```
Though this is complicated because our analytics strategy current includes both pageviews (useful for both filesets and works) and downloads (useful only for filesets, though these numbers could be summed for Work-level download stats).
```

:arrow_heading_up: implies works have page views and filesets have downloads, right? Sorry if I misunderstood.

**Comment by awead** [comment]
I guess it isn't. I was just agreeing with him.

**Comment by mjgiarlo** [comment]
OK, just checking!

**Comment by hectorcorrea** [comment]
Are we not going to have views for FileSets (in addition to downloads)?

**Comment by mjgiarlo** [comment]
This is part of #1455.

**Comment by cam156** [comment]
This ticket assumes that the work page and the file set page are the same graph as we currently have, but just displaying rolled up information gathered in #1455 for the work, and individual information for the fileset.

**Comment by mjgiarlo** [comment]
@cam156 I believe where we landed in #1455 is that every FileSet will have its views and downloads tracked, and every GenericWork will have its views tracked, and all of this will be stored in the application database (as before). I :heart: the idea of rolling up stats in each GenericWork *also* so that you can look at cumulative analytics, though I'm inclined to say that in particular might be a good candidate for a post-7.0.0 release that can build what's already stored in the DB. Reasonable?

**Comment by mjgiarlo** [comment]
Are you still working on this ticket, @njaffer, or is it available for someone else to pick up?

**Comment by njaffer** [comment]
@mjgiarlo: I think this can be closed by #1535 and #1536

**Comment by mjgiarlo** [comment]
Whoever takes this ticket on, please verify this ticket can be closed. (The FileSet show page should have a link to analytics, and that link should work.)

**Comment by atz** [comment]
Link is there, works:
![screen shot 2016-04-06 at 4 11 56 pm](https://cloud.githubusercontent.com/assets/109954/14335727/66f1d07c-fc12-11e5-89f2-dc454e203dc6.png)

**Comment by mjgiarlo** [comment]
:clap:
